src/flisp/julia_extensions.c

```diff
@@ -84,7 +84,7 @@ static int is_wc_cat_id_start(uint32_t wc, utf8proc_category_t cat)
 
               (wc >= 0x266f &&
                (wc == 0x266f || wc == 0x27d8 || wc == 0x27d9 || // ♯, ⟘, ⟙
-                (wc >= 0x27c0 && wc <= 0x27c2) ||  // ⟀, ⟁, ⟂
+                (wc >= 0x27c0 && wc <= 0x27c2) ||  // ⟀, ⟁
```

Comment:
You only changed the comment, not the code.  Need to change the upper limit to `0x27c1`.